lib/ansible/modules/windows/win_package.ps1

```diff
@@ -326,10 +327,12 @@ if ($state -eq "absent") {
 
             if ($program_metadata.msi -eq $true) {
                 # we are uninstalling an msi
-                $temp_path = [System.IO.Path]::GetTempPath()
-                $log_file = [System.IO.Path]::GetRandomFileName()
-                $log_path = Join-Path -Path $temp_path -ChildPath $log_file
-                $cleanup_artifacts += $log_path
+                if ( -Not $log_path ) { 
+                  $temp_path = [System.IO.Path]::GetTempPath()
```

Comment:
Are you able to keep the indentation the same as the other parts of this script (4 spaces)

Reply:
sure